src/entries/knockout-ui.ts

```diff
@@ -63,10 +63,11 @@ export * from "../knockout/components/panel/panel";
 export * from "../knockout/components/popup/popup";
 export * from "../knockout/components/progress/buttons";
 export * from "../knockout/components/progress/progress";
-export * from "../knockout/components/template-renderer";
+export * from "../knockout/components/template-renderer/template-renderer";
 export * from "../knockout/components/title-actions/title-actions";
-export * from "../knockout/components/string-editor";
-export * from "../knockout/components/string-viewer";
+export * from "../knockout/components/string-editor/string-editor";
+export * from "../knockout/components/string-viewer/string-viewer";
+export * from "../knockout/components/logo-image/logo-image";
```

Comment:
As for me to create a folder for two files that already are grouped by the same name is too much...

Reply:
My opinion that separate files in separate folders is good and predictable. But just my opinion